### ISSUE TITLE
Rely on stable builds rather than dev builds ...

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/ansible#8]
Once geOrchestra 14.12 is released, this will require us to add 2 more jobs in CI.

**Comment by landryb** [comment]
Well, i was maybe thinking that the master branch of the playbook was targetting the master branch of georchestra, and "eventually" doing a branch for releases.. but my initial main goal has always been 'deploy a test instance of georchestra master' :)

**Comment by fvanderbiest** [comment]
Since we (docker & ansible recipes) are overwriting conf files directly inside the webapps, relying on master is extremely fragile. It may break everytime.

I would advise to setup a 14.12 branch and make it the default in your repository, yes.

**Comment by fvanderbiest** [comment]
But yes, let's wait we setup 2 more CI jobs ...

**Comment by fvanderbiest** [comment]
Since Terranis will rely on this playbook to deploy stable 15.12 to 6 instances, we'll very soon need the option to deploy stable packages rather than master.

**Comment by fvanderbiest** [comment]
Oh and BTW http://paulla-build.georchestra.org/maven/ is 403 forbidden :-(

**Comment by pmauduit** [comment]
> Oh and BTW http://paulla-build.georchestra.org/maven/ is 403 forbidden :-(

There is no need to let the whole document root browseable, maven is fully capable of finding the dependencies without activating any autoindex capabilities

**Comment by fvanderbiest** [comment]
> There is no need to let the whole document root browseable, maven is fully capable of finding the dependencies without activating any autoindex capabilities

Nice :-)

**Comment by pmauduit** [comment]
The Debian repository on paulla-build already targets 15.12 FYI (master still to be done)

**Comment by fvanderbiest** [comment]
...which means this patch would make the playbook deploy 15.12 rather than master:
```
diff --git a/playbooks/georchestra.yml b/playbooks/georchestra.yml
index 5d9d3c7..371c371 100644
--- a/playbooks/georchestra.yml
+++ b/playbooks/georchestra.yml
@@ -51,8 +51,8 @@
         gitversion: master
       },
       debian: {
-        repo: "deb https://build.georchestra.org/debian jessie main",
-        key: "https://build.georchestra.org/debian/build.georchestra.org.asc"
+        repo: "deb http://193.55.222.230/debian jessie main",
+        key: "http://193.55.222.230/debian/build.georchestra.org.asc"
       }
     }
     geonetwork: {
```
To be tested...

Thanks Pierre :-)

**Comment by fvanderbiest** [comment]
Hmm, there's something wrong: it tries to fetch http://193.55.222.230/debian/pool/main/g/georchestra-catalogapp/georchestra-catalogapp_16.06+201603251032_all.deb while only http://193.55.222.230/debian/pool/main/g/georchestra-catalogapp/georchestra-catalogapp_15.12%2b48_all.deb is available

**Comment by pmauduit** [comment]
> Hmm, there's something wrong: it tries to fetch http://193.55.222.230/debian/pool/main/g/georchestra-catalogapp/georchestra-catalogapp_16.06+201603251032_all.deb while only http://193.55.222.230/debian/pool/main/g/georchestra-catalogapp/georchestra-catalogapp_15.12%2b48_all.deb is available



> +        repo: "deb http://193.55.222.230/debian jessie main",
> +        key: "http://193.55.222.230/debian/build.georchestra.org.asc"

it is not jessie anymore, but 15.12 or master on the new server

**Comment by fvanderbiest** [comment]
Nice, provisioning again ...

**Comment by landryb** [comment]
Rather than a note in the README, i'd rather have branches in the ansible git repo, 15.12 (or stable ?) to deploy 15.12, master to deploy master...

**Comment by landryb** [comment]
Unfortunately, we cant use https:// on 193.55.222.230, since for now the certificate only points to sdi-stable.georchestra.org :) One more subdomain to add to the letsencrypt config..

**Comment by fvanderbiest** [comment]
WIP on https://github.com/pmauduit/ansible/commits/18.06

**Comment by landryb** [comment]
technically this one is fixed by the presence of the 19.04 branch (even if it sort of works)
